compose/ui/ui/src/jsWasmMain/kotlin/androidx/compose/ui/draganddrop/DragAndDrop.jsWasm.kt

```diff
@@ -21,14 +21,14 @@ import androidx.compose.ui.geometry.Offset
 /**
  * A representation of an event sent by the platform during a drag and drop operation.
  */
-actual class DragAndDropEvent
+actual class DragAndDropEvent(val offset: Offset)
```

Comment:
I'd rather keep it minimal for now. What separates drag and drop event s from the rest of browser events we process in my opinion, that it's highly unlikely that compose consumer will ever need anything browser-specific in case of drag events. Drag and Drop is basically about accessing dragImage and dragData, both of this will be addressed. 

Apart from that,  mouse events and keyboard events (and touch events for that matter) have more or less one-to-once conceptual correspondence with compose events. But DOM drag and drop events is merely a helper, an inner implementation that I honestly would prefer users not to have an access to as long as it possible.

Reply:
The explanation makes sense to me. We can add it later if needed, but it's more difficult to remove something later.